utils.py

```diff
@@ -0,0 +1,59 @@
+import logging
+from os.path import expanduser, join
+
+import yaml
+from dask import config
+from dask.distributed import Client
+from dask_jobqueue import PBSCluster, SGECluster, SLURMCluster
+
+def dask_scheduler(jobs):
```

Comment:
You could change this to accept the yaml file or a dictionary with the parameters you'd like to pass.